ballerina-tests/tests/34_return_record_bidi_streaming_server.bal

```diff
@@ -26,11 +26,9 @@ listener grpc:Listener ep34 = new (9124);
 service "HelloWorld34" on ep34 {
 
     remote isolated function sayHello34(stream<SampleMsg34, error?> clientStream) returns stream<SampleMsg34, error?>|error? {
-        io:println("Connected sucessfully.");
-        check clientStream.forEach(isolated function(SampleMsg34 val) {
+        checkpanic clientStream.forEach(isolated function(SampleMsg34 val) {
```

Comment:
```suggestion
        check clientStream.forEach(isolated function(SampleMsg34 val) {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/2ef479d6607ef5aa5336331221c12c698cdd94cb